Code/Support/RestKit-Prefix.pch

```diff
@@ -5,6 +5,11 @@
 #ifdef __OBJC__
     #import <Foundation/Foundation.h>
 
+// Fix for compilation on iOS 6 where this is not defined.
+#if !defined NS_DESIGNATED_INITIALIZER
```

Comment:
You are totally right - you do need to make sure that the code quality is maintained.

You should not get this confused with personal preference of code layout, however, especially when there is no technical difference.

If you want to bring up the issue of being polite, a simple thank you for finding a fix and creating a pull request is a great place to start.